.php-cs-fixer.php

```
@@ -13,6 +13,28 @@
     [
         '@PSR12' => true,
         'declare_strict_types' => true,
+        // These four put back what canonical printing removes. `doctor` names them, because
+        // without them a normalisation loses layout that a rule could have restored.
+        'declare_parentheses' => true,
+        'blank_line_after_opening_tag' => true,
```

PHPStan (2.2.8)

[error] Array has 2 duplicate keys with value 'blank_line_after_opening_tag' ('blank_line_after_opening_tag', 'blank_line_after_opening_tag').


(array.duplicateKey)